packages/strapi-design-system/src/Select/Select.js

```diff
@@ -149,7 +150,7 @@ export const Select = ({
 
   return (
     <Field hint={hint} error={error} id={generatedId}>
-      <Stack spacing={label || hint || error ? 1 : 0}>
+      <Stack spacing={label || hint || typeof error === 'string' ? 1 : 0}>
```

Comment:
```suggestion
      <Stack spacing={label || hint || hasStringError ? 1 : 0}>
```

Reply:
Maybe you can also set the spacing like that but it has to be tested:

```suggestion
      <Stack spacing={label || ariaDescribedBy ? 1 : 0}>
```

Reply:
no because _ariaDescribedBy_ value is
`const ariaDescribedBy = errorMessageId || hintMessageId;`

and _hintMessageId_ value is
`hint ? `${generatedId}-hint` : undefined`

but in the condition we need hint and not the _hintMessageId_ value